src/components/Island.css

```diff
@@ -1,5 +1,6 @@
 .Island {
   --padding: 0;
+  word-break: keep-all;
```

Comment:
could you add this only to `ShortcutKey` element?

https://github.com/excalidraw/excalidraw/blob/master/src/components/ShortcutsDialog.tsx#L88-L93

Reply:
Already edited.